add industries grid component

jesicasusanto opened the pull request:
Resolves Issue #7 
This PR 
- implements the industries grid component of the web page that is responsive to the screen size
- fixed the zoomed in screen when first opening the web page

- Before :

<img width="960" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/a34c5fbb-f21b-4fdf-9e88-0c0f211bb50b">

<img width="930" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/d18a3e00-90e4-4be0-b9d3-6228ab7b1cfb">


- After :


<img width="959" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/645d72f9-a02e-43eb-ad7e-3a2eb70bb0bd">

<img width="960" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/4955f061-57b2-4c03-bb71-cd19d644b2f9">


steps to reproduce 
same as [Getting Started](https://github.com/OpenAdaptAI/web#getting-started) in readme

jesicasusanto commented:
Made the cards responsive to screen size
Before :
<img width="538" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/d6714601-f07a-4a83-9393-f65364cdd066">

After :
<img width="533" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/d751de2c-a772-4d28-887f-edd437001edb">

jesicasusanto commented:
Change cards color to be dark purple then add card shadows.
Before :
<img width="939" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/9bf54d2d-8812-4cdf-ac6f-11c27a8a4891">
After :
<img width="922" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/17ed7d26-abb9-4a66-98b2-99e88fe761d0">

Would love to get feedback on this @abrichr . Thank you!

dianzrong commented:
@jesicasusanto I really like the shadows! Do you think the card colours should be lighter to improve visibility?

jesicasusanto commented:
@dianzrong thank you! i agree too. I've made the grids background lighter.

jesicasusanto commented:
Ready for review @abrichr @Jonathan-Sendsen.

Summary :
- added industry grids
- added value proposition

abrichr commented:
Please add "(For Developers)" after "Getting Started"

abrichr commented:
Please modify grid styles so that padding is symmetrical horizontally/vertically between items

abrichr commented:
@jesicasusanto can you please add an icon to each one, e.g. from https://thenounproject.com/ ? 🙏

abrichr commented:
https://openadapt.ai/ :

<img width="652" alt="image" src="https://github.com/OpenAdaptAI/web/assets/774615/0f338734-7398-4d65-9102-6f0a6ea02e85">

https://deploy-preview-15--cosmic-klepon-3c693c.netlify.app/:

<img width="762" alt="image" src="https://github.com/OpenAdaptAI/web/assets/774615/547f45eb-01a2-4ac4-badb-7014852dc31a">


Can you please restore the padding / margins around "AI for Humans", "Automate your workflows", "No programming required"?

abrichr commented:
Also please feel free to implement https://github.com/OpenAdaptAI/web/issues/17 while you're at it 😄

jesicasusanto commented:
Hi @abrichr , thank you for the feedback.
I have made the changes according to your feedback :
- restored the padding/margins
- restored the Get Started button and link it to industries grid
- made the buttons secondary
For the logos, I feel like we need them to be white logos which is only included in the paid package from https://thenounproject.com/. Will this be okay with you? Please let me know what you think! Thank you.

abrichr commented:
For each of the industries, please mention specific use cases / roles / applications. You can find these in the issues.

e.g.:
> Optimize patient care and medical record keeping by integrating our automation tool, freeing healthcare professionals for critical tasks.

This should mention specifics, e.g.:

> Automate Revenue Cycle Management, Auditing, and Encounter Documentation in Cerner, Epic, and Excel, and more.

Please assemble a list of these descriptions in a Google Doc that we can collaborate on. Include links to specific Github comments and other sources 🙏

abrichr commented on components/IndustriesGrid.js:
```diff
@@ -0,0 +1,113 @@
+      buttonLabel: 'Join Waitlist',
+      logo: '/images/noun-developer.svg',
+    },
+  ];
```
Excellent thank you for defining these as objects! 🙏

jesicasusanto commented:
Hi @abrichr for cropping the logos to content, gimp doesn't support exporting the logo back to an svg file. As an alternative, I adjusted the width and height attributes inside the svg files according to the heightxwidth sized given by the gimp "crop to content" command. Please let me know what you think of this solution. Thank you! 
Before :

<img width="290" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/008c3474-8cdb-4aa2-9126-a4eae9d332f5">

After :
<img width="284" alt="image" src="https://github.com/OpenAdaptAI/web/assets/87709055/5ada5efb-d886-46a3-bfcf-8978540d49d6">

abrichr commented:
Thanks @jesicasusanto !

Can you please remove the developer grid item? Then modify the "Getting Started" header to "For Developers".

jesicasusanto commented:
Update @abrichr 
- [x] remove the developer grid item (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/00b66e13d933d4e89e9b2fe5fb71994861bbc5b4)
- [x] modify the "Getting Started" header to "For Developers" (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/c54e26ced9749e56aa6ff32f5bbcab8e4e9d1604)
- [x] remove this heading altogether:
Revolutionizing Industries with AI-First Automation for Growth and Transformation. (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/0f44787c392cb6d6c36b6d749f6b174d12dab0fa)
- [x] rename "Freight" to "Logistics" (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/3eae0bfe3f224e4f821b8851ea27f5c238b5520f).
- [x] center grid content vertically (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/d52ac596e2cf02818c92559954f1e8fa761d2521)

I also added a silver border around the cards when we hover them as you suggested on our last sprint in https://github.com/OpenAdaptAI/web/pull/15/files#diff-4130362d983b52ff81285caab923910466ada416e0ca9974ad2ffd36f674366e:~:text=%7D-,.card%3Ahover%20%7B,-border%3A%202 . But Im not sure if this aligns with your vision for the hover card animation. Would love to get a clarification in this. Looking forward to your feedback!

abrichr commented:
Hi @jesicasusanto :

- It looks like the hover animation on the icons is no longer working. Can you please fix it?
- The border is too bright. Please make it much more subtle. It should be barely perceptible. It should also be animated.
- The background color of the grid component does not appear to change on hover. It should transition to a slightly brighter color over e.g. 0.5s.

Please let me know if anything is unclear!

abrichr commented:
@jesicasusanto the updated border color looks better, and the background color animation looks good 👍 

Todo:
- fix icon animation (make sure you are opening the page in incognito mode and/or clearing your cache to reproduce the issue. Right now icons just switch between small and large with no animation)
- set the "Join Waitlist" border color to transparent on hover rather than removing it altogether. This is necessary to prevent the rest of the grid content from moving down when the user hovers with their mouse due to the reduced size from the border being removed.

Please let me know if anything is unclear!

jesicasusanto commented:
Update @abrichr :

- [x] added margin around logo (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/9b212d65ad1bd93b222b31d37d2d3a704bc84245)
- [x] added outline when the industry grid is hovered (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/302fab44c67eb70db1938730c26edcbf8f94dd85)
